java/java.source.base/src/org/netbeans/modules/java/source/save/CasualDiff.java

```diff
@@ -5004,7 +5009,31 @@ private int diffSince(DCDocComment doc, DCSince oldT, DCSince newT, int[] elemen
         }
         return elementBounds[1];
     }
-    
+//    
```

Comment:
Why is this commented out?  Is it worth keeping these comments or not?

Reply:
since its a private method, I think it should be either removed or simply left there (if there are plans to use it later). 

Commenting big code blocks out with `//` is not optimal for version control, `/**/` would be better since it produces less merge conflicts due to easier automated diff.